### PR TITLE
Making a variable a void* will make things better

### PR DESCRIPTION
**Pull request by jacekczerwonka**
Void* is a more generic type than Task so seems more appropriate.

### CODE REVIEW

**Comment by jacekczerwonka** [comment]
Hello @paigerodeghero, thank you for participating in my first pull request. I think I'll need some help getting it accepted.

**Comment by paigerodeghero** [comment]
Hi @jacekczerwonka, yes I can help you make this pull request successful.  I'm not exactly sure what you are doing here, could you edit the first message to explain what you are trying to achieve?

**Comment by jacekczerwonka** [comment]
I changed the description. Thanks @paigerodeghero.

File: api/pictapi.cpp [hunk 1]
```diff
@@ -7,6 +7,7 @@ using namespace pictcore;
 
 //
 // forward declarations
+// these are necessary in C++
```
Comment (arfon) [inline api/pictapi.cpp]: @paigerodeghero @jacekczerwonka - it might be a good idea to add a little more context here? Perhaps a link to docs?

**Comment by margaretstorey** [comment]
What the?  The description doesn't make sense to me...

File: api/pictapi.cpp (hunk 1 above)
Comment (jacekczerwonka) [reply api/pictapi.cpp]: I agree. I will make the change here.

Comment (paigerodeghero) [reply api/pictapi.cpp]: Great! :)

**Comment by arfon** [comment]
source

**Comment by jacekczerwonka** [comment]
Thank you @paigerodeghero for your amazing support. And to the @YODAbot for facilitating and looking out for me.

**Pull request merged**